lib/view/drawing_canvas/widgets/canvas_side_bar.dart

```diff
@@ -1,10 +1,12 @@
-// ignore: avoid_web_libraries_in_flutter
-import 'dart:html' show AnchorElement;
+import 'dart:async';
+import 'dart:io';
 import 'dart:ui' as ui;
+import 'dart:ui';
```

Comment:
1. `dart:ui` is already imported, please reuse the previous one.

Reply:
I just updated this